DEVELOPER.md

```diff
@@ -351,18 +351,53 @@ The e2e tests use the [kuttl](https://github.com/kudobuilder/kuttl/) testing fra
    make run-int-tests | tee kuttl.out
    ```
 
-### Run Individual Tests
+### Run Individual e2e Tests
 
 You can run individual tests from the command line with the [KUTTLE CLI](https://kuttl.dev/docs/cli.html#setup-the-kuttl-kubectl-plugin).
 
-> **NOTE**
-> Before you can run an individual test, you need to set up a communal endpoint. To set up a [MinIO](https://min.io/) endpoint, run the following make target:
+Prerequisite steps:
+
+1. Before you can run an individual test, you need to set up a communal endpoint. To set up a [MinIO](https://min.io/) endpoint, run the following make target:
 >
 > ```shell
 > make setup-minio
 > ```
 >
-> To set up a different communal endpoint, see [Custom communal endpoints](#custom-communal-endpoints).
+    To set up a different communal endpoint, see [Custom communal endpoints](#custom-communal-endpoints).
```

Comment:
The explanation is currently inside a shell frame. We need to adjust the format. Could you try to remove the symbol ">" from lines 361 to 365? If this works, do the same thing for other env vars below.

Reply:
I made the change. It turns out ">" is used to add an indent to all lines in a graph.